src/main/java/karuna/karuna_backend/receiver/domain/ReceiverMapper.java

```diff
@@ -0,0 +1,21 @@
+package karuna.karuna_backend.receiver.domain;
+
+import karuna.karuna_backend.receiver.dto.ReceiverDTO;
+import karuna.karuna_backend.receiver.dto.ReceiversDTO;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+class ReceiverMapper {
+
+    static ReceiversDTO mapToDto(List<Receiver> receivers) {
+        return new ReceiversDTO(receivers.stream().map(Receiver::getEmail).collect(Collectors.toSet()));
```

Comment:
Correct me if I am wrong but here you are creating DTO for Receivers including ONLY email. But below you use Id aswell in DTO.